src/utils/youtube.js

```diff
@@ -7,3 +13,63 @@ export function createYoutubeEmojiNode(emote) {
   newNode.setAttribute('data-emoji-id', emote.id);
   return newNode;
 }
+
+export function getLiveChat() {
+  return document.getElementsByTagName('yt-live-chat-renderer')[0]?.__data?.data;
+}
+
+const MAX_EPHMERAL_MESSAGE_DEPTH = 100;
+let YoutubeMessageListener = null;
+function unmountExpiredEphemeralMessage() {
+  const chatItems = document.querySelector(CHAT_ITEMS_SELECTOR);
+  if (chatItems == null) {
+    return;
+  }
+  const visibleItems = chatItems?.__dataHost?.__data?.visibleItems;
+  if (visibleItems == null || visibleItems.length < MAX_EPHMERAL_MESSAGE_DEPTH) {
+    return;
+  }
+  let hasEphemeralMessage = false;
+  chatItems.__dataHost.__data.visibleItems = visibleItems.filter((item, index, items) => {
```

Comment:
rather than leverage their list, it may be more reliable to keep our own list and check for its index in the dom children instead. this way, if they do remove our entry we can still clean up